core/domain/feedback_jobs_continuous.py

```diff
@@ -201,7 +214,8 @@ def _get_continuous_computation_class(cls):
 
     @classmethod
     def entity_classes_to_map_over(cls):
-        return [feedback_models.FeedbackThreadModel]
+        return [feedback_models.FeedbackThreadModel,
```

Comment:
I thought of trying to do that too. But it is slightly different (using realtime models, which I need to read up about). Basically I am not sure of what strategy we should use to generalize the continuous jobs. Also I am not sure, but I think, even without these jobs, the feedback threads can work (basic functionality to message to a thread, etc) for other entities. So I decided to leave it for later.

Reply:
As an FYI, you can read up about realtime models in core/jobs.py. See the documentation for BaseContinuousComputationManager.

Reply:
Let's leave the job name as is -- just be really clear in the docstrings that it is specific to exploration feedback and not a general job. This is because it's a continuous computation that's currently running and we don't want errors to occur in the next release just because we forget to stop and start it.

Reply:
Ok. Thanks!